docs/nuget-org/nuget-org-faq.yml

```diff
@@ -104,15 +104,12 @@ sections:
     questions:
       - question: How to recover NuGet.org password login?
         answer: |
-          Please note that the [NuGet.org Password login has been discontinued](https://blog.nuget.org/20180515/NuGet.org-will-only-support-MSA-AAD-starting-June.html) and the only way to log in to NuGet.org is with a personal Microsoft account (MSA) or Azure Active Directory (AAD) account. However, in case you are unable to access your associated MSA/AAD accounts you might need to use password login for recovering your NuGet.org account. In this situation follow the steps below.
-          - **Requirement:** You will need to have access to the email that is associated with the account for which you need to recover the password.
-          - Go to the [Forgot password page](https://www.nuget.org/account/ForgotPassword)
-          - Enter the **email** address that is associated with the NuGet.org account you wish to recover.
-          - Click the **Send** button.
-          - You will get an email to the specified email address account with a link to reset your password. Click on this link and set the new password. If you can't find the mail check your "junk" folder.
-          - Once done, you can now login with username/password on NuGet.
-          - To login with username/password, use the **Sign in using Nuget.org account** link on the  [NuGet.org login page](https://www.nuget.org/users/account/LogOn).
+          Please note that the [NuGet.org Password login has been discontinued](https://blog.nuget.org/20180515/NuGet.org-will-only-support-MSA-AAD-starting-June.html). Unfortunately, NuGet.org has no way to recover password accounts, you can only log in to NuGet.org with a personal Microsoft account (MSA) or Azure Active Directory (AAD) account. 
           
+          If you want to transfer the ownership of your packages from your previous account, please follow our [dispute resolution process](https://docs.microsoft.com/en-us/nuget/nuget-org/policies/dispute-resolution).
```

Comment:
Dispute resolution is confusing to link here, I think. I understand what you're getting at but from the user perspective this isn't a dispute with another person, it is a support flow. Maybe I am misunderstanding?

Reply:
Also, nit: links to other docs page have a special relative syntax which should be used.

Reply:
Added something similar to what we have in our support flow which is `Please let us know if you need assistance in transferring ownership of your packages to a different account by sending an email to [support@nuget.org](mailto:support@nuget.org).`.